keras/engine/topology.py

```diff
@@ -1229,6 +1229,45 @@ def count_params(self):
                                    self.name + '.build(batch_input_shape)`.')
         return sum([K.count_params(p) for p in self.weights])
 
+    def count_flops(self):
```

Comment:
@fchollet I would argue in favor of having op counting in `model.summary()`.  

Op counts are a good predictor of speed, and can be very unevenly distributed among layers (so it is not obvious where to optimize).  People tend to optimize what they can see, optimizing param counts has been very easy with keras, op counts / speed not so much until now.

Also, most papers seem to report both param counts and op counts as a measure of the "size" of the network.

If you are stongly against having op counting as default in model.summary(), perhaps there could be an option to display it.  I would really like it to be the default though, with an option to hide it.